src/flake8_simplify/plugins/ast_bool_op.rs

```diff
@@ -143,6 +145,29 @@ pub fn duplicate_isinstance_call(checker: &mut Checker, expr: &Expr) {
     }
 }
 
+define_violation!(
+    pub struct AAndNotA(String);
+);
+
+impl Violation for AAndNotA {
+    fn message(&self) -> String {
+        let name = &self.0;
+        format!("Use `False` instead of `{name} and not {name}`")
+    }
+
+    fn autofix_available(&self) -> bool {
+        true
+    }
+
+    fn autofix_description(&self) -> Option<String> {
+        Some("Replace with `False`".to_string())
+    }
+
+    fn placeholder() -> Self {
+        Self("...".to_string())
+    }
+}
```

Comment:
We could almost certainly auto-generate these too for the initial migration.

Reply:
Yes that's what I meant by:

> this conversion would probably best be automated via a single-use Python script

Reply:
Ah sorry, I thought this was referring specifically to the code-to-kind mapping.

Reply:
No worries :)